src/kv/KeyValueDB.h

```diff
@@ -409,6 +409,13 @@ class KeyValueDB {
 				       const std::string& key_prefix) {
     return 0;
   }
+  /// estimate space utilization for a specified range (in bytes)
+  virtual int64_t estimate_range_size(
+    const std::string& prefix,
+    const std::string& key_from,
+    const std::string& key_to) {
+      return 0;
```

Comment:
(a question: why providing a default (null) implementation here (and to the other functions in this interface)? i.e. - why isn't this an abstract class?)